<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Datetime</class>
 <widget class="QWidget" name="Main_Datetime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QPushButton
{
font: 16px;color: rgb(192,192,192);background-color: rgb(0, 0, 0);border: 1px solid white;border-top: 2px solid white;border-left: 2px solid white;
}
QLabel
{
color:rgb(255,255,255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>721</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>721</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CCU的时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>系统的时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_CCU_date">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>90</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: yellow;</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_CCU_time">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>120</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: yellow;</string>
   </property>
   <property name="text">
    <string>0.00.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_SYS_date">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>90</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: yellow;</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_SYS_time">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>120</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: yellow;</string>
   </property>
   <property name="text">
    <string>0.00.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string>年份设定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Set_Year">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color: yellow;</string>
   </property>
   <property name="text">
    <string>2017</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Set_Hour">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color: yellow;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string>小时设定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Set_Month">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color: yellow;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string>月份设定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Set_Minute">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color: yellow;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string>分钟设定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Set_Day">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color: yellow;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string>日期设定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Set_Second">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color: yellow;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string>秒设定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Year_Dec">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Year_Asc">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Month_Dec">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>3</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Month_Asc">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>4</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Day_Dec">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>5</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Day_Asc">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>6</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Hour_Dec">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>7</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Hour_Asc">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>8</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Minute_Des">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>9</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Minute_Asc">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>10</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Second_Dec">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>11</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Second_Asc">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>12</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Update">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>407</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>0</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid white;
background-color: rgb(0, 0, 0);
font:18px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>时间更新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
